tests/test_qtest_proxies.py

```diff
@@ -1,10 +1,10 @@
 import pytest
 
-from pytestqt.qt_compat import USING_PYSIDE
+# noinspection PyUnresolvedReferences
```

Comment:
Yes, otherwise PyCharm complains this is not being used. It's was complaining because the mark is now "eval"ued instead of being explicit.